code/game/objects/structures/petrified_statue.dm

```diff
@@ -80,12 +80,16 @@
 		return FALSE
 	var/obj/structure/statue/petrified/S = new(loc, src, statue_timer)
 	S.name = "statue of [name]"
-	bleedsuppress = 1
+	bleedsuppress = TRUE
 	S.copy_overlays(src)
 	var/newcolor = list(rgb(77,77,77), rgb(150,150,150), rgb(28,28,28), rgb(0,0,0))
 	S.add_atom_colour(newcolor, FIXED_COLOUR_PRIORITY)
+	addtimer(CALLBACK(src, .proc/unpetrify), statue_timer)
 	return TRUE
 
+/mob/living/carbon/human/proc/unpetrify()
+	bleedsuppress = FALSE
+
```

Comment:
I take it statues don't have a "become not statue" proc you could hook into

Reply:
Yup, the code is pretty old, it just deletes the statue at the end

Reply:
Oh, turns out there was one that I just didn't see. Updated it to use that just now.